tools/build_defs/scala/scala.bzl

```diff
@@ -95,6 +108,11 @@ def _collect_jars(ctx):
       jars += target.java.transitive_runtime_deps
   return jars
 
+def _args_for_suites(suites):
+  args = ["-o"]
+  for suite in suites:
+    args.extend(["-s", suite])
```

Comment:
Certainly, here it is: https://groups.google.com/forum/#!msg/bazel-dev/5Kbbwr11XOA/znTY-H1DBgAJ

It's also in the top-level description of the change but maybe it's more visible here inline.